### PR TITLE
authemtication pages

### PR DESCRIPTION
**Pull request by anupkgurung**
login
signin

### CODE REVIEW

File: page/login.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                        <label for="password">Password</label>
+                        <a class="link-style-none forgot-password">Forgot Password</a>
+                    </div>                   
+                    <input class="input-width" autocomplete="off" type="password" name="password" id="password" placeholder="password">
+                </div>
+                <div class="remember-credentials-check flex-row padding-top-2">
+                    <input type="checkbox" id="remember" class="checkbox">
+                    <label for="remember">Remember Me</label>
+                </div>
+                <div class="">
```
Comment (aniketsingh13) [inline page/login.html:75]: remove class

**Review by aniketsingh13 — APPROVED** [review]
look good to me

**Review by maitrakhatri — APPROVED** [review]
looks perfect to me. Just a suggestion, you can use the `required` attribute in the input fields so that in future you don't need JS or edit html files again.

File: page/login.html (hunk 1 above)
Comment (anupkgurung) [reply page/login.html:75]: need to use that :)

**Comment by anupkgurung** [comment]
> looks perfect to me. Just a suggestion, you can use the `required` attribute in the input fields so that in future you don't need JS or edit html files again.

cool !!

**Pull request merged**